alwaysontop/main.js

```diff
@@ -38,27 +56,55 @@ function onAlwaysOnTopWindow(
         const win = event.newGuest = new BrowserWindow(
             Object.assign(options, {
                 backgroundColor: 'transparent',
-                width: SIZE.width,
-                height: SIZE.height,
                 minWidth: SIZE.width,
                 minHeight: SIZE.height,
-                maxWidth: SIZE.width,
-                maxHeight: SIZE.height,
                 minimizable: false,
                 maximizable: false,
-                resizable: false,
+                resizable: true,
                 alwaysOnTop: true,
                 fullscreen: false,
                 fullscreenable: false,
                 skipTaskbar: true,
                 titleBarStyle: undefined,
                 frame: false,
                 show: false
-            }, getPosition())
+            }, getPosition(), getSize())
         );
         win.once('ready-to-show', () => {
             win.showInactive();
         });
+
+        //for macOS we use the built-in setAspectRatio on resize, for other we use custom implementation
+        if (os.type() === 'Darwin') {
```

Comment:
Why did we move the resize code from the render to the main process?

Reply:
as it seemed to be more fluent on main process